README.md

```diff
@@ -53,10 +54,35 @@ the two Raspberry Pi architectures:
 
 ## Building
 
-There two options for building the gadget snap: cross building and native
-building.
+There are two general approaches to building the pi-gadget snap: _managed_ and
+_manual managed_.
 
-### Cross building
+The easiest managed approach is to simply run `snapcraft` within the root of
+this repository on a classic Ubuntu installation, such as an amd64-based Ubuntu
+server or desktop, or even an arm64-based Ubuntu running on a Raspberry Pi.
+Snapcraft will create either a [Multipass](https://multipass.run/) or
+[LXD](https://linuxcontainers.org/lxd/introduction/) build environment and
+produce the gadget snap automatically.
+
+Another managed option is to run `snapcraft remote-build`. This command
+offloads the snap build process to the [Launchpad build
+farm](https://launchpad.net/builders), pushing the potentially
+multi-architecture snap back to your machine when the build completes. See
+[Remote build](https://snapcraft.io/docs/remote-build) for further details.
+
+Managed build environments will mirror the distro series declared in the `base`
+setting of the gadget's snapcraft.yaml, such as _core20_ or _core18_.
+
+Manually managed builds include building the gadget snap on Ubuntu Core running
+on a Raspberry Pi, for example, and systems where you want to first manually
+isolate a build from the host system. In both cases, you first manually
+create an LXD instance from which you can run `snapcraft --destructive-mode` to
+use the instance as the build environment.
+
+Examples of both a _managed_ build and a _manually managed_ build are outlined
+below.
+
+### Example managed build
```

Comment:
just a comment re-reading the below section again, while managed is the most convenient method, it is not the most performant, the most performant is manually managed since there is not all the extra snapcraft overhead of managing the VM/container, since you are managing the stuff yourself when building

Reply:
Thanks for taking a look! I've tweaked the text slightly to remove this.